deployment/services/cloudflare-security.ts

```diff
@@ -14,23 +15,26 @@ function toExpressionList(items: string[]): string {
 }
 
 export function deployCloudFlareSecurityTransform(options: {
-  envName: string;
+  environment: Environment;
   ignoredPaths: string[];
   ignoredHosts: string[];
 }) {
   // We deploy it only once, because CloudFlare is not super friendly for multiple deployments of "http_response_headers_transform" rules
   // The single rule, deployed to prod, covers all other envs, and infers the hostname dynamically.
-  if (options.envName !== 'prod') {
+  if (!options.environment.isProduction) {
     console.warn(
-      `Skipped deploy security headers (see "cloudflare-security.ts") for env ${options.envName}`,
+      `Skipped deploy security headers (see "cloudflare-security.ts") for env ${options.environment.envName}`,
     );
+
     return;
   }
 
   const expression = `not http.request.uri.path in { ${toExpressionList(
     options.ignoredPaths,
   )} } and not http.host in { ${toExpressionList(options.ignoredHosts)} }`;
 
+  // TODO: When Preflight PR is merged, we'll need to change this to build this host in a better way.
+  const labHost = `lab-worker.${options.environment.rootDns}`;
```

Comment:
Is `lab-worker` the final decided name? What were other options?

Reply:
we can do whatever you pick :)

Reply:
at the moment it's `lab-worker` in @dimaMachina's PR